packages/node/tsconfig.json

```diff
@@ -2,7 +2,7 @@
   "compilerOptions": {
     "strict": true,
     "esModuleInterop": true,
-    "lib": ["ES2020"],
+    "lib": ["ES2020", "DOM"],
```

Comment:
What is DOM used for here?

Reply:
This is necessary for the `WebAssembly` type. If we remove it, we get the following typescript errors:

```
src/edge-wasm-plugin.ts(22,46): error TS2503: Cannot find namespace 'WebAssembly'.
src/edge-wasm-plugin.ts(22,46): error TS4055: Return type of public method from exported class has or is using private name 'WebAssembly'.
src/edge-wasm-plugin.ts(24,35): error TS2503: Cannot find namespace 'WebAssembly'.
src/edge-wasm-plugin.ts(29,33): error TS2304: Cannot find name 'WebAssembly'.
```